[BUG] make docker:boot failing on mainline

**Describe the bug**
Running `make docker:boot` fails during the build of acpica.

**To Reproduce**
Steps to reproduce the behavior:
1. Checkout mainline
2. Run `make docker:boot`
3. Observe the following compilation errors: 
```bash
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsinit.c:152:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
```

**Expected behavior**
`make docker:boot` successfully builds acpica

**Logs**
```bash
$ make docker:boot
Creating docker image
Sending build context to Docker daemon  12.78MB
Step 1/5 : FROM ubuntu:20.04
 ---> ba6acccedd29
Step 2/5 : RUN apt-get update -y
 ---> Using cache
 ---> 928a632a0d6d
Step 3/5 : RUN apt-get install -y gcc make xorriso qemu-utils qemu qemu-system-x86 patch
 ---> Using cache
 ---> 94cd72d69956
Step 4/5 : RUN DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install grub2 kmod python
 ---> Using cache
 ---> c8a35679f189
Step 5/5 : CMD ["/bin/bash"]
 ---> Using cache
 ---> 576ff8c254e5
Successfully built 576ff8c254e5
Successfully tagged ktf:build
running target 'boot' in docker
CC  drivers/acpi/acpica/osl.o
CC  drivers/acpi/acpica/source/components/namespace/nsload.o
CC  drivers/acpi/acpica/source/components/namespace/nsxfname.o
CC  drivers/acpi/acpica/source/components/namespace/nswalk.o
CC  drivers/acpi/acpica/source/components/namespace/nsnames.o
CC  drivers/acpi/acpica/source/components/namespace/nspredef.o
CC  drivers/acpi/acpica/source/components/namespace/nsinit.o
CC  drivers/acpi/acpica/source/components/namespace/nsdumpdv.o
CC  drivers/acpi/acpica/source/components/namespace/nsrepair.o
CC  drivers/acpi/acpica/source/components/namespace/nsprepkg.o
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsload.c:152:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nswalk.c:152:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nspredef.c:154:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
CC  drivers/acpi/acpica/source/components/namespace/nsutils.o
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsnames.c:152:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsload.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsdumpdv.c:153:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsnames.o] Error 1
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsinit.c:152:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsxfname.c:155:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsrepair.c:152:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsrepair.o] Error 1
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsinit.o] Error 1
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nspredef.o] Error 1
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsxfname.o] Error 1
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nswalk.o] Error 1
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsdumpdv.o] Error 1
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsutils.c:153:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/source/components/namespace/nsprepkg.c:152:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsutils.o] Error 1
make: *** [Makefile:201: drivers/acpi/acpica/source/components/namespace/nsprepkg.o] Error 1
In file included from /home/connojd/dev/ktf/drivers/acpi/acpica/source/include/acpi.h:164,
                 from drivers/acpi/acpica/osl.c:37:
/home/connojd/dev/ktf/drivers/acpi/acpica/source/include/platform/acenv.h:309:10: fatal error: acktf.h: No such file or directory
  309 | #include "acktf.h"
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:201: drivers/acpi/acpica/osl.o] Error 1
make: *** [docker:boot] Error 2
```

**Additional context**
I believe the issue stems from the use of this line in the ACPICA_INSTALL assignment in the Makefile: `$(SYMLINK) $(ACPICA_DEST_DIR)/acktf.h $(ACPICA_DEST_DIR)/source/include/platform/acktf.h)`. According to [this](https://stackoverflow.com/questions/31881904/docker-follow-symlink-outside-context) symlinks are not supported for passing through to the container

--- a/Makefile
+++ b/Makefile
@@ -79,6 +79,7 @@ SHELL := bash
 RM := rm
 LN := ln
 SYMLINK := $(LN) -s -f
+HARDLINK := $(LN) -f
 OBJCOPY := objcopy
 STRIP := strip
 ifeq ($(SYSTEM), MACOS)
@@ -135,7 +136,7 @@ ifeq ($(CONFIG_ACPICA),y)
 ACPICA_INSTALL := $(shell [ -d $(ACPICA_DEST_DIR)/source ] ||                         \
                           $(TAR_CMD_ACPICA) $(ACPICA_TARBALL) $(ACPICA_UNTAR_DIRS) && \
                           $(PATCH) -p0 < $(ACPICA_PATCH) &&                           \
-                          $(SYMLINK) $(ACPICA_DEST_DIR)/acktf.h $(ACPICA_DEST_DIR)/source/include/platform/acktf.h)
+                          $(HARDLINK) $(ACPICA_DEST_DIR)/acktf.h $(ACPICA_INCLUDE)/platform/acktf.h)
 endif
 
 SOURCES     := $(shell find . -name \*.c)

